packages/webdriverio/webdriverio.d.ts

```diff
@@ -37,6 +44,12 @@ declare namespace WebdriverIO {
          * to this callback will be returned to the client.
          */
         executeAsync: (script: string | ((...arguments: any[]) => void), ...arguments: any[]) => Promise<any>;
+
+        mockCalls(mockId: number): Promise<void>;
+
+        mockRequest(url: string, filterOptions: MockFilterOptions): { mockId: number };
+
+        respondMock(mockId: number, payload: object): Promise<void>;
```

Comment:
Is this the right way to go, adding the missing types here? If yes, any tips how to document them?

Reply:
Hummm... looks like this is not the right approach? Some tests failed in another module... 🤔

Reply:
Outdated, figured out already